airflow/providers/google/cloud/operators/kubernetes_engine.py

```diff
@@ -305,11 +303,9 @@ def __init__(
         *,
         location: str,
         cluster_name: str,
-        use_internal_ip: bool = False,
```

Comment:
This is going to break backwards compat!

Reply:
Replace it with optional parameter and add deprecation message